### PR TITLE
Flutter plugin link updated.

### PR DESCRIPTION
**Pull request by theyashjani**
To configure Flutter plugin in Android Studio the link has been updated

I have replaced the Flutter plugin download link for Android Studio IDE it was misplaced by Visual Studio Code plugin Link.

By updating the link, Beginners will get correct information to download the Flutter plugin in Android Studio IDE.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by yashjani01** [comment]
@sfshaza2 any updates?

**Comment by amanv8060** [comment]
Just curious, the plugin is compatible with both Android Studio and Intellij, so why is the separation actually required?

File: README.md [hunk 1]
```diff
@@ -89,7 +89,8 @@ Information on how to get started can be found in our
 [Hot reload animation]: https://github.com/flutter/website/blob/main/src/assets/images/docs/tools/android-studio/hot-reload.gif?raw=true
 [Hot reload]: https://flutter.dev/docs/development/tools/hot-reload
 [Visual Studio Code]: https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
-[IntelliJ / Android Studio]: https://plugins.jetbrains.com/plugin/9212-flutter
+[IntelliJ]: https://plugins.jetbrains.com/plugin/9212-flutter
+[Android Studio]: https://flutter.dev/docs/development/tools/android-studio
```
Comment (Hixie) [inline README.md:93]: should this point to https://docs.flutter.dev/get-started/editor?tab=androidstudio instead?

**Comment by Hixie** [comment]
cc @jacob314 to check if this change is reasonable.

File: README.md (hunk 1 above)
Comment (yashjani01) [reply README.md:93]: Yes

**Comment by jacob314** [comment]
> plugin

I don't understand

> cc @jacob314 to check if this change is reasonable.

I believe this pull request should be closed as I believe the fix made by https://github.com/flutter/flutter/commit/32122480b5cff211cb6cbe545d22d143af121f25 resolves the issue this pull request was addressing while making it clearer that one Flutter plugin supports both IntelliJ and Android Studio.

Adding a link to https://docs.flutter.dev/get-started/editor?tab=androidstudio from the readme could be reasonable but I would keep in separate from the links pointing to the plugins for VSCode and Android Studio and I would add a link to https://docs.flutter.dev/get-started/editor?tab=vscode as well for consistency.

**Comment by yashjani01** [comment]
@jacob314 no doubt but if user/newbie want to setup Flutter in Android Sfudio it should redirect it to Android Studio setup page, and if the user want to setup flutter on VS Code the individual link should redirect to VS Code specific page.
With the help of two different links the user will setup the System with ease rather than going and searching for other.

**Comment by jacob314** [comment]
@yashjani01 I support adding a link for Android Studio setup instructions as well as a link to the Android Studio/ IntelliJ plugin. The problem is right now this CL has an IntelliJ link that goes to the plugin for both Android Studio and IntelliJ and an Android Studio link that goes to the android studio setup instructions. I would suggest renaming the link labeled [IntelliJ] to be `[Flutter IntelliJ / Android Studio Plugin]` to reduce confusion. Right now it is not clear why the one link is to the plugin while the other is to setup instructions.

**Comment by yashjani01** [comment]
@jacob314 I'll rename it..

**Comment by yashjani01** [comment]
@jacob314 Changes are successfully done.

**Comment by Hixie** [comment]
@yashjani01 Is this still a PR you are interested in? If you merge the conflicts and apply the suggested edits I can land it.

**Comment by yashjani01** [comment]
> @yashjani01 Is this still a PR you are interested in? If you merge the conflicts and apply the suggested edits I can land it.

Yes, I'll do today.

**Comment by Hixie** [comment]
I'm going to close this PR for now, since it's been a while with no updates, please if you are still interested in fixing this please submit a new PR. Thanks!

**Comment by theyashjani** [comment]
**@guidezpl please add cla i think it is been removed by you on => ** "[guidezpl] removed the [cla: yes](https://github.com/flutter/flutter/labels/cla%3A%20yes) label [on May 1, 2022](https://github.com/flutter/flutter/pull/91418#event-6528974477)"

**Pull request closed without merging**